### PR TITLE
add babelify instructions for using canjs with es6 and browserify

### PR DESCRIPTION
**Pull request by cherifGsoul**

### CODE REVIEW

**Comment by matthewp** [comment]
lgtm

File: docs/can-guides/experiment/setting-up-canjs.md [hunk 1]
```diff
@@ -213,15 +213,54 @@ Next, change your `package.json` to include the required `stringify` configurati
 Next, run Browserify from the command line:
 
 ```
-./node_modules/browserify/bin/cmd.js -t stringify main.js > bundle.js
+./node_modules/browserify/bin/cmd.js -t stringify main.js > dist/bundle.js
+```
+
+By default Browserify works with CommonJS modules (with `require()` statements). To use it with ES6 modules (with `import` statements like in the guides), configure the `babelify` plugin in your `packagae.json`:
```
Comment (phillipskevin) [inline docs/can-guides/experiment/setting-up-canjs.md]: `packagae.json` -> `package.json`

Comment (cherifGsoul) [reply docs/can-guides/experiment/setting-up-canjs.md]: You changed it?

Comment (phillipskevin) [reply docs/can-guides/experiment/setting-up-canjs.md]: @chasenlehara took care of it.

Comment (cherifGsoul) [reply docs/can-guides/experiment/setting-up-canjs.md]: thank you :)

**Pull request merged**